src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -137,11 +132,54 @@ internal static void ParseResponseObject(IDictionary<string, object> responseObj
             }
         }
 
+        internal static void TryParseStatusCode(IDictionary<string, object> responseObject, ref HttpStatusCode statusCode)
+        {
+            object statusValue;
+
+            if (!responseObject.TryGetValue("statusCode", out statusValue, ignoreCase: true) &&
+                !responseObject.TryGetValue("status", out statusValue, ignoreCase: true))
+            {
+                return;
+            }
+
+            if (statusValue is int ||
+                statusValue is HttpStatusCode
+            )
+            {
+                statusCode = (HttpStatusCode) statusValue;
+                return;
+            }
+
+            if (statusValue is long ||
+                statusValue is short ||
+                statusValue is sbyte ||
+                statusValue is ulong ||
+                statusValue is uint ||
+                statusValue is ushort ||
+                statusValue is byte)
+            {
```

Comment:
Not done yet :wink:
(int)(object)200l wont work but (int)(long)(object)200l will so can do that instead of convert.

Sure I can remove  byte and sbyte.